pkg/systemd/services.html

```diff
@@ -56,13 +59,27 @@
         <button class="btn btn-default" role="tab" data-pattern="\.timer$" translatable="yes">Timers</button>
         <button class="btn btn-default" role="tab" data-pattern="\.path$" translatable="yes">Paths</button>
       </div>
-      <button id="create-timer" data-toggle="#timer-dialog" data-target="#timer-dialog" class="btn btn-primary pull-right" translate>Create Timer</button>
+      <div>
```

Comment:
There's an empty `<div>` here. Usually this comes up with React, where `<React.Fragment>` should be used. I'm not sure why there's an empty div here though. If you need to group elements in a wrapper, try to use a semantic element (when it makes sense) and please _at least_ give it a class and/or ID regardless to describe why this is here. (Something like "services-filter" might work.)